Makefile

```diff
@@ -547,6 +550,17 @@ endif
 	$(verbose) echo "override MCS_CATS := $(MCS_CATS)" >> $(headerdir)/build.conf
 	$(verbose) $(INSTALL) -m 644 $(support)/Makefile.devel $(headerdir)/Makefile
 
+########################################
+#
+# Build policy interface database
+#
+build-interface-db: install-headers
+	$(verbose) $(SEPOLGEN_IFGEN) $(VERBOSE_FLAG) -i $(headerdir)
+
+rebuild-interface-db:
+	$(verbose) rm -rf $(headerdir)
+	$(verbose) $(MAKE) build-interface-db
```

Comment:
Wouldn't something like this work, without having to recursively call this makefile:
```
build-interface-db: install-headers $(tmpdir)/load

rebuild-interface-db $(tmpdir)/load:
	$(verbose) $(SEPOLGEN_IFGEN) $(VERBOSE_FLAG) -i $(headerdir)
	@touch $(tmpdir)/load
```
If something in headerdir needs to be deleted, my preference would be not to delete everything.

Reply:
This does not seem to be addressed.

Reply:
The initial design for the targets was:
  - `build-*`: generate the interface - always run `sepolgen-ifgen`, so calling the target twice generates the database twice
  - `rebuild-*`: generate the interface but assure that no leftover interfaces are in the header directory (after a re-factorization, removal of an module ...)

But since Refpolicy takes backward-compatibility serious the reasons for the `rebuild-*` target probably do not apply.

Dropped the `rebuild-*` target.